app/routes.php

```diff
@@ -3,21 +3,12 @@
 
 use App\Controllers\CoursesAPIController;
 use Slim\App;
-use Slim\Views\PhpRenderer;
-use Slim\Interfaces\RouteCollectorProxyInterface as Group;
+
 
 return function (App $app) {
+    //could get rid of this line? not sure what it's doing
```

Comment:
I'm pretty sure this is what gets our dependencies...

Reply:
probably,  php storm greys out $container but maybe that's because it doesn't like slim

Reply:
Yeah I think it's a Slim thing. Is that what they call a magic method?

Reply:
it is now 😅